src/shared/components/FilterList/InjectionOption.tsx

```diff
@@ -43,10 +43,10 @@ const FluxInjectionOption: FC<Props> = ({
   setHoverdFunction,
 }) => {
   // TODO: move this to a flag once design wants to branch it out further
```

Comment:
✂️ this comment - doesn't look like it's referring to anything meaningful anymore

Reply:
Nice catch. 
Will cut it in a follow up PR. (Want to merge this before any conflicts merge. 😅 )